src/app/components/FinanceV2Components/CardRow/index.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+
 import { StyledCardRow } from './styled';
 
 interface ICardRowProps {
```

Comment:
I would adjust the interface to accept `className` and use it in the top div. This way, we could adjust the component and add `tw-mt-2` in both of the usages in this story because it looks a bit weird without a top margin (we use `tw-mt-2` in Portfolio page as well).